InlineColumnConfiguration/InlineColumnConfiguration.php

```diff
@@ -22,10 +22,23 @@ public function register() {
         $this->description = plugin_lang_get("description");
 
         $this->version = "1.0";
-        $this->requires = array(
-			"MantisCore" => "1.2.6",
-			"jQueryUI" => "1.8"
-        );
+
+		$t_version_core = substr(
+				MANTIS_VERSION,
+				0,
+				strpos( MANTIS_VERSION, '.', strpos( MANTIS_VERSION, '.' ) + 1 )
+			);
+		if( $t_version_core==='1.3' ) {
+			$this->requires = array(
+				"MantisCore" => "1.3"
+			);
+		}
+		else {
+			$this->requires = array(
+				"MantisCore" => "1.2.6",
+				"jQueryUI" => "1.8"
+			);
+		}
```

Comment:
Considering the differences between the 2 versions, I'm personally not a big fan of maintaining compatibility like this (I'd rather have 2 distinct branches)

That being said, if it has to be done this way, why not simply use 
```
if( version_compare( MANTIS_VERSION, '1.3', '<' ) ) {
	# 1.2.x
} else {
	# 1.3.x or higher
}
```